Inventory.py

```diff
@@ -1,24 +1,27 @@
-from flask import Flask, render_template, url_for, request
+from flask import Flask, render_template, url_for, request, redirect
 import sqlite3
 
-#connection to DB
-conn = sqlite3.connect('Inventory.db')
-
-#setup cursor object
-cursor = conn.cursor()
```

Comment:
doing this was throwing an error for me about accessing sqlite from the wrong thread, moved the connection into the methods

Reply:
I did receive a database locked issue; perhaps this was the culprit and I misunderstood the traceback?